matcher/encoder.py

```diff
@@ -8,20 +8,23 @@
 import numpy as np
 import logging
 
+
 def _score_to_cost(score, scaling_factor=100):
-    '''
+    """
```

Comment:
it seems you are making some changes in the code style, are you following any conventions?

Reply:
These are mostly changes suggested by my editor. Ideally, we should add a linter such as `flake8` as a pre-commit or pre-push hook. This will help with consistent formatting and syntax across the repo.

Reply:
instead of having a hook, can we ask the user that commit the code to follow the guidelines?

Reply:
Created a separate PR #222 for this.